package/base-files/files/bin/config_generate

```diff
@@ -160,18 +160,19 @@ generate_network() {
 
 	case "$protocol" in
 		static)
-			local ipad
+			local ipad netm prefix
 			case "$1" in
 				lan) ipad=${ipaddr:-"192.168.1.1"} ;;
 				*) ipad=${ipaddr:-"192.168.$((addr_offset++)).1"} ;;
 			esac
 
 			netm=${netmask:-"255.255.255.0"}
```

Comment:
I know you didn't touch this, but the quotes are in a weird place.  I would have done:

```
netm="${netmask:-255.255.255.0}"
```

instead.

Reply:
I agree, however using shellcheck reveals multiple similar issues and other problems at the same level, which makes me question whether we actually need to go down that rabbit hole since it normally just works and making it shellcheck-compliant would require quite a few changes unrelated to the purpose of this PR.